### PR TITLE
Reduce dmesg verbosity

### PR DESCRIPTION
**Pull request by plbossart**
We have too many logs, so start reducing the verbosity.

This work also exposes a problem with SoundWire interrupts. For some reason, even when PARITY interrupts are not enabled, codecs still report them on startup. By default all masks should be disabled on a hardware reset, not sure what's going on here but this adds too many false positives.

@shumingfan @bardliao @RanderWang can you look into this?

### CODE REVIEW

File: drivers/soundwire/bus.c [hunk 1]
```diff
@@ -1047,14 +1047,16 @@ static int sdw_initialize_slave(struct sdw_slave *slave)
 	int ret;
 	u8 val;
 
+	dev_err(slave->bus->dev,
+		"programming SDW_SCP_INTMASK1\n");
```
Comment (lyakh) [inline drivers/soundwire/bus.c]: I'm a bit confused: first this doesn't seem to belong to this PR. And as a side note for this commit: this error message would confuse me, if I saw it in my kernel log. And it's supposed to appear every time SDW is used?

Comment (plbossart) [reply drivers/soundwire/bus.c]: This is to expose a problem with the other log, that shows whether we program the parity or not. It's not final of course, I just want to root-cause why we see parity errors for no good reason.

**Comment by plbossart** [comment]
removed last commit and moved it to PR #1984

**Comment by kv2019i** [comment]
SOFCI TEST

**Comment by kv2019i** [comment]
One chekcpatch warning  for spelling, but that's an upstream patch, so let's ignore that.

Device test shows some errors triggered by lack of traces (test case expectes verbose traces to be enabled). Sent a mail to CI folks separately about this. Let's give a bit of time for discussing and then merge. Seems good to go otherwise.

**Comment by plbossart** [comment]
> One chekcpatch warning for spelling, but that's an upstream patch, so let's ignore that.
> 
> Device test shows some errors triggered by lack of traces (test case expectes verbose traces to be enabled). Sent a mail to CI folks separately about this. Let's give a bit of time for discussing and then merge. Seems good to go otherwise.

@xiulipan @wenqingfu please provide a PR for sof-test that shows when the CI/QA devices add dynamic debug on their own instead of assuming the kernel enables it.This will help btw test other kernel branches that don't enable dynamic debug with Makefile changes

**Comment by xiulipan** [comment]
@plbossart I think a kmod reload w/o `/etc/modprobe.d/sof-dyndgb.conf` file will have different logs
@aiChaoSONG Can we have a new test case for the dynamic debug logs in sof-test?
issue to track this task:  https://github.com/thesofproject/sof-test/issues/173

**Comment by kv2019i** [comment]
Ok, thanks, all, merging.
